lib/Stats.js

```diff
@@ -1,24 +1,55 @@
 /*
-	MIT License http://www.opensource.org/licenses/mit-license.php
-	Author Tobias Koppers @sokra
+MIT License http://www.opensource.org/licenses/mit-license.php
+Author Tobias Koppers @sokra
 */
 "use strict";
 
 const RequestShortener = require("./RequestShortener");
 const SizeFormatHelpers = require("./SizeFormatHelpers");
 const formatLocation = require("./formatLocation");
 const identifierUtils = require("./util/identifier");
+const Tapable = require("tapable").Tapable;
+const SyncHook = require("tapable").SyncHook;
+const HookMap = require("tapable").HookMap;
+const SyncWaterfallHook = require("tapable").SyncWaterfallHook;
 
 const optionsOrFallback = (...args) => {
 	let optionValues = [];
 	optionValues.push(...args);
 	return optionValues.find(optionValue => typeof optionValue !== "undefined");
 };
 
-class Stats {
+const isTypeOf = (variable, type) => {
+	if (typeof variable === type) return true;
+	return false;
```

Comment:
nit: You could return `typeof variable === type`. I think it would improve the readability.